src/state/application/updater.ts

```diff
@@ -42,6 +48,27 @@ export default function Updater(): null {
     [chainId, setState]
   )
 
+  const chainConnectivityWarningActive = useAppSelector((state) => state.application.chainconnectivityWarning)
```

Comment:
consider moving this to its own custom hooks similar to `useQueryCacheInvalidator` for readability.